content/administration/install/deploy.rst

```diff
@@ -388,20 +388,109 @@ The solutions to support livechat/motifications in a WSGI application are:
   with ``/longpolling/`` to
   :option:`the longpolling port <odoo-bin --longpolling-port>`.
 
+.. _streaming:
+
+Serving Static Files and Attachments
+====================================
```

Comment:
New guidelines require titles and headings to be in sentence case
https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#titles-and-headings

Headings in this doc aren't consistent, anyway... so 😕